spydra/src/test/java/com/spotify/spydra/submitter/runner/SubmissionCliParserTest.java

```diff
@@ -59,4 +57,13 @@ public void testJobIdMaxLength() throws IOException {
     assertEquals(88, fixed_id.length());
   }
 
+
+  @Test
+  public void testLabels() throws IOException {
+    SubmissionCliParser parser = new SubmissionCliParser();
+    String[] args = {"--labels=k1=v1", "--labels=k2=v2,k3=v3"};
+    SpydraArgument argument = parser.parse(args);
```

Comment:
Code in the call to compute is never tested and behaves odd in my opinion. Could you ensure converse of that?

Reply:
Adding tests.